### DISCUSSION TITLE
Multi projet

### DISCUSSION THREAD

**Discussion by Arnaud-Askee** [dbarzin/mercator#739] [Ideas]
Hello,
L'application permet de faire la cartographie d'un seul contexte.
Dans mon cas, j'interviens pour plusieurs entreprises et j'aimerai pouvoir réaliser la cartographie de plusieurs projets.

Merci :)

**Comment by dbarzin** [comment]
Il est possible d'installer plusieurs instances de Mercator sur un même serveur avec des URL ou ports et bases de données distincts.
Cela demande des changements du fichier .ENV pour changer le nom de la base de données utilisée et de créer un virtual host par instance de Mercator.
